velox/vector/arrow/Bridge.cpp

```diff
@@ -38,10 +38,17 @@ static constexpr size_t kMaxBuffers{3};
 // carried by ArrowArray.private_data
 class VeloxToArrowBridgeHolder {
  public:
-  VeloxToArrowBridgeHolder() {
-    for (size_t i = 0; i < kMaxBuffers; ++i) {
+  VeloxToArrowBridgeHolder() = default;
+
+  void resizeBuffers(size_t bufferCount) {
```

Comment:
can we add a comment here that when you call this function you may need to re-acquire the underlying buffer (`getArrowBuffers()`)?

Reply:
Done